portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -10,9 +10,27 @@
 @WebServlet("/hello")
```

Comment:
Can the servlet name be changed to something more descriptive like get-quote?

Reply:
Hi Sophie, @kuanyiwu
Thanks for your feedback! I have updated the path to "/quotes".